arch/x86/core/Kconfig.intel64

```diff
@@ -20,9 +20,9 @@ config TEST_EXTRA_STACKSIZE
 config SYSTEM_WORKQUEUE_STACK_SIZE
 	default 8192
 
-config EXCEPTION_STACK_SIZE
+config X86_EXCEPTION_STACK_SIZE
```

Comment:
There is also ARMV7_EXCEPTION_STACK_SIZE

Do we want to have a Kconfig for each architecture? Any reason why this should not be moveds up to arc/Kconfig and set defaults per architecture and where it is supported?

Reply:
a lot of arches don't even have an exception stack.
we only use on on x86_64.
ARM has a whole pile of stacks specific to particular exceptions.
It's really the wild west, I think it's mostly futile to try to create arch-neutral definitions, even the ISR stack size means different things across arches.